x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/TSDBRestEsqlIT.java

```diff
@@ -0,0 +1,188 @@
+                },
+                "k8s": {
+                    "properties": {
+                        "pod": {
+                            "properties": {
+                                "uid": {
+                                    "type": "keyword",
+                                    "time_series_dimension": true
+                                },
+                                "name": {
+                                    "type": "keyword"
+                                },
+                                "cpu": {
+                                    "properties": {
+                                        "limit": {
+                                            "type": "scaled_float",
+                                            "scaling_factor": 1000.0,
+                                            "time_series_metric": "gauge"
+                                        },
+                                        "nanocores": {
+                                            "type": "long",
+                                            "time_series_metric": "gauge"
+                                        },
+                                        "node": {
+                                            "type": "scaled_float",
+                                            "scaling_factor": 1000.0,
+                                            "time_series_metric": "gauge"
+                                        }
+                                    }
+                                },
+                                "network": {
+                                    "properties": {
+                                        "rx": {
+                                             "type": "long",
+                                            "time_series_metric": "gauge"
+                                        },
+                                        "tx": {
+                                            "type": "long",
+                                            "time_series_metric": "gauge"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        """;
+
+    private static final String SETTINGS = """
+        {
+             "index": {
+                 "mode": "time_series",
+                 "routing_path": [
+                     "metricset",
+                     "k8s.pod.uid"
+                 ]
+             }
+         }
+        """;
+
+    private static final String BULK =
+        """
+            {"create": {}}
+            {"@timestamp": "2021-04-29T17:29:12.470Z", "metricset": "pod", "k8s": {"pod": {"name": "cat", "uid":"947e4ced-1786-4e53-9e0c-5c447e959507", "network": {"tx": 2001818691, "rx": 802133794},"cpu": {"limit": 0.3787411612903226, "nanocores": 35222928, "node": 0.048845732}}}}
```

Comment:
I wonder if we'd be better of doing a `doc("cat", "947e4ced-1786-4e53-9e0c-5c447e959507", 2001818691, 802133794)` or something. It's nice that this is standard copy from somewhere but it might be easier to read if we use a method to stamp it out.